api/api.go

```diff
@@ -998,9 +998,6 @@ func (api *Server) getBlockMetasByHeader(height uint64) (*iotextypes.BlockMeta,
 	if err != nil {
 		return nil, status.Error(codes.NotFound, err.Error())
 	}
-	if err := api.getBlockMetaUpgrade(header.Height()); err != nil {
```

Comment:
After removing this, shall we handle the error here: https://github.com/iotexproject/iotex-core/blob/89483c317149092ea2adcd7335c45344c423539f/api/api.go#L988-L989